### DISCUSSION TITLE
MC Board update 00.00.22.41

### DISCUSSION THREAD

**Discussion by Roulendo** [X1Plus/X1Plus#193] [Q&A]
I have installed X1plus 1.1. After a successful boot, I receive the message: 'This component's firmware version does not match the running firmware version.' I am then asked if I want to install firmware version 00.00.22.41. When I agree to proceed, I receive an error message: 'Downloading version xxxx from Bambulab failed: Download failed (remote server returned error code 0).' What could be wrong? Could someone please give me some advice? Thank you :)

Edit:
I think I've identified the problem. It was related to the Wi-Fi connection. After I changed the Wi-Fi channel on the router, everything is working again and the firmware for the motherboard is on the SD card. Should I proceed with the update? I want to keep my jailbreak.

**Comment by mkosmo** [comment]
I updated and haven't had any problems, but I'm not sure what I'd actually need to look for after an MC update.

**Reply by Irondrgntp** [reply]
I updated after seeing this notification and now I cannot boot the custom firmware.
![20240421_090336](https://github.com/X1Plus/X1Plus/assets/129942141/e2e66a79-167d-4244-8384-ad340003601e)
Can anyone help? How can I fix the install? I see no options to retract this.

**Reply by mkosmo** [reply]
I imagine it'll take connecting to the UART to get any more diagnostics.  Can you boot to OEM at least?

**Reply by Roulendo** [reply]
@Irondrgntp, how did you solve the problem? Is your custom firmware working again after the MC board update? What was the issue? I'm still on firmware version 1.0 because the MC board update 00.00.22.41 seems too risky to me. It would be really nice if someone could comment on this. Should we proceed with the MC board update 00.00.22.41? Thank you very much

**Reply by Irondrgntp** [reply]
I had to wipe the SD and reinstall a few times before everything booted as it should. I thought it was updating the MC that caused the issue but now I'm not sure because it also happened after messing with lan/lan shield/cloud settings. Regardless, it all works fine now.

**Reply by Roulendo** [reply]
@Irondrgntp, thank you for your feedback. I will also proceed with the MC Board update 00.00.22.41, as it seems to cause no issues. It's unfortunate that the CFW documentation contains no information about this update.

**Comment by Roulendo** [comment]
It would have been helpful if the firmware developers had provided additional information about the update notification for the release of the X1Plus 1.1. I assume that everyone who updates to version 1.1 receives this MC Board update message. I have reverted to version 1.0 because I did not dare to proceed with the update.
